webextension/scripts/utils.js

```diff
@@ -3,7 +3,7 @@
 let isArray = (a) => (!!a) && (a.constructor === Array)
 let isObject = (a) => (!!a) && (a.constructor === Object)
 
-let isFirefox = (navigator.userAgent.indexOf('Firefox') !== -1)
+let isFirefox = (navigator.userAgent.indexOf('Firefox') > -1)
```

Comment:
Why did you change this ?

Reply:
I was checking out MDN web docs documents and I found this: https://developer.mozilla.org/en-US/docs/Web/API/Window/navigator
They used ` > -1` so, I made the change.

Reply:
Would you like me to revert it? :-)

Reply:
No. You may create another variable and check for edge browser. Please ask Owenl to create new host name for edge browser. !
@cgorringe may need to check this once ! Thank you !

Reply:
Yes, we need `edge-api`, and I will add the check for that. Thank you!

Reply:
We may want to rethink how we use isFirefox etc... as we'll have multiple browsers to support, including Edge and Safari.

Using `!== -1` vs `> -1` doesn't really make any difference. I'd leave it at `!== -1`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf

Reply:
I reverted this change. Thank you!